### PR TITLE
Support .lottie files in LottieGen.

### PR DESCRIPTION
**Pull request by simeoncran**
See https://dotlottie.io for file details.
LottieGen will now read .lottie files as well as .json files.

### CODE REVIEW

File: LottieGen/FileProcessor.cs [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.IO;
+using System.IO.Compression;
+using System.Linq;
+using Microsoft.Toolkit.Uwp.UI.Lottie.DotLottie;
+
+/// <summary>
+/// Processes a file containing an animation and produces various generated outputs.
```
Comment (simeoncran) [inline LottieGen/FileProcessor.cs:12]: Can contain more than one animation.

File: LottieGen/FileProcessor.cs [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
+            case ".lottie":
+                return ProcessDotLottieFile(
+                    options: _options,
+                    reporter: _reporter,
+                    filePath: _filePath,
+                    outputFolder: _outputFolder,
+                    timestamp: _timestamp);
+
+            case ".json":
+            default:
```
Comment (derobinsMSFT) [inline LottieGen/FileProcessor.cs:68]: It's not an error to have a file that doesn't end in `.json` or `.lottie`?

**Review by derobinsMSFT — APPROVED** [review]
:shipit:

File: LottieGen/FileProcessor.cs (hunk 2 above)
Comment (simeoncran) [reply LottieGen/FileProcessor.cs:68]: Yeah, on purpose. Nothing requires JSON files to be .json. However .lottie files do need to be named with .lottie.

---
In reply to: [512253491](https://github.com/windows-toolkit/Lottie-Windows/pull/376#discussion_r512253491) [](ancestors = 512253491)

**Pull request merged**